src/components/molecules/Visualizer/Engine/Cesium/Marker/index.tsx

```diff
@@ -131,9 +131,19 @@ const Marker: React.FC<PrimitiveProps<Property>> = ({ layer }) => {
   );
 
   const pixelOffset = useMemo(() => {
-    const padding = 15;
-    const x = (img?.width && style == "image" ? img.width : pointSize) / 2 + padding;
-    const y = (img?.height && style == "image" ? img.height : pointSize) / 2 + padding;
+    const padding = imageSize ? 15 : 40;
+    const x =
+      (img?.width && imageSize && (!style || style === "image")
+        ? img.width * imageSize
+        : pointSize) /
+        2 +
+      padding;
+    const y =
+      (img?.height && imageSize && (!style || style === "image")
+        ? img.height * imageSize
+        : pointSize) /
+        2 +
+      padding;
```

Comment:
```suggestion
    const x =
      (img?.width && (!style || style === "image")
        ? img.width * (imageSize ?? 1)
        : pointSize) /
        2 +
      padding;
    const y =
      (img?.height && (!style || style === "image")
        ? img.height * (imageSize ?? 1)
        : pointSize) /
        2 +
      padding;
```

Does this not work?

Reply:
In this suggestion when we upload a new image the imageSize will be undefined by default that is why (imageSize ?? 1) will always be 1 at the beginning.
however when we upload the image, the size of the image is not relevant to the imageSize and all images will be the same size regarding of the imageSize

![Screen Shot 0004-06-03 at 11 46 41 AM](https://user-images.githubusercontent.com/35835415/171776948-a45d0f13-1157-467e-b883-403e3f35985b.png)

and then when we change the imageSize manually it will adjust to the actual imageSize

![Screen Shot 0004-06-03 at 11 47 44 AM](https://user-images.githubusercontent.com/35835415/171777026-16d1a67a-d6ae-4282-8662-31af41935109.png)

so in your suggestion we will have something like this at the start before adjusting the imageSize manually

![Screen Shot 0004-06-03 at 11 52 27 AM](https://user-images.githubusercontent.com/35835415/171777452-a1eb43f6-a204-4970-9c75-bed87d552626.png)

Reply:
I see. You mean `imageSize` will not be undefined thanks to `const { imageSize = 0.1 } = property?.default ?? {};`.

So in your expression,

```
const padding = imageSize ? 15 : 40;
```

I think `padding` should be always `15` unless `0` is intentionally specified for `imageSize`. Is it incorrect?

Reply:
Not exactly what I meant.
because I have not added `const{ imageSize = 0.1 } = property?.default ?? {};` to the code base yet.
what I mean is the padding will always be 15 unless image size is 0 or undefined(at the beginning).